db/warehouse_structure.sql

```diff
@@ -115,116 +115,116 @@ CREATE TYPE public.record_type AS ENUM (
 
 CREATE FUNCTION public.service_history_service_insert_trigger() RETURNS trigger
     LANGUAGE plpgsql
-    AS $$

```

Comment:
This keeps happening in my db file whenever I run migrations. I will revert manually, but does this happen to anyone else..?